src/CharNames.java

```diff
@@ -8,87 +9,84 @@
      */
 
 
-        private Map<String, Set> charCatalog;
+    private Map<String, Set> charCatalog;
 
-        /**
-         * Constructor tat creates an empty map
-         */
-        public CharNames()
-        {
-            charCatalog = new HashMap<>();
-        }
+    /**
+     * Constructor tat creates an empty map
+     */
+    public CharNames() {
+        charCatalog = new HashMap<>();
```

Comment:
technically don't have to instantiate the map here you can drop the constructor all together